benchmarks/build.gradle.kts

```diff
@@ -1,31 +1,17 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to you under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright (c) 2020, PostgreSQL Global Development Group
```

Comment:
This should be 2023 right?

I'm not an expert on these things, but I think you'd need to leave the original notice place and add ours under it to cover subsequent changes.

Reply:
fair point. Ya, we can't remove licenses

Reply:
> This should be 2023 right?

The copyright date should be "the first date the file was introduced".
In other words, we claim the file is part of pgjdbc since 2020.

Reply:
> leave the original notice place and add ours under it to cover subsequent changes.

The original notice was "apache 2" by mistake as it was automatically added, and I did not notice the header was wrong.